Dockerfile

```
@@ -27,17 +27,17 @@
 #  will take a bit of work.
 
 # Build will be where we build the go binary
-FROM artifactory.algol60.net/csm-docker/stable/registry.suse.com/suse/sle15:15.5 as build
+FROM artifactory.algol60.net/csm-docker/stable/registry.suse.com/suse/sle15:15.6 as build
```

To stop the complaint message about mismatched case in the build logs.

```suggestion
FROM artifactory.algol60.net/csm-docker/stable/registry.suse.com/suse/sle15:15.6 AS build
```